### PR TITLE
Add 2020 version of Reproducible Science in RStudio

### PR DESCRIPTION
**Pull request by aays**
- Code demo dramatically shortened
- Blurbs added for Git and Packrat

### CODE REVIEW

**Comment by QuLogic** [comment]
You should update `lesson.md` to link this in some way.

File: lessons/r/reproducible-rstudio/lesson-2020.Rmd [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+
+Note: material for an extended version of this lesson, delivered in the Spring
+2020 semester, can be found [here][lesson-old]. This version is truncated and
+intended to be delivered in 40-50 minutes online, thus de-emphasizing the
+code-along sections.
+
+## What does it mean to work reproducibly in RStudio?
+
+Reproducible computational science endeavours to make both the data used in
+analyses as well as the code used to process this data both available and
```
Comment (QuLogic) [inline lessons/r/reproducible-rstudio/lesson-2020.Rmd]: You have 'both' both before and after the two things.

File: lessons/r/reproducible-rstudio/lesson-2020.Rmd [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+Packrat is a package management system for R that allows users to associate
+specific versions of packages with projects. This means that a project can
+contain its own self contained packages forever, which is especially useful if
+these packages eventually get updated in ways that break compatibility (since
+Packrat will specifically allow for the maintenance of whichever version of a
+given package was used). More can be found [here][packrat].
+
+
+
+[happy-git-r]: happygitwithr.com
```
Comment (QuLogic) [inline lessons/r/reproducible-rstudio/lesson-2020.Rmd]:
```suggestion
[happy-git-r]: https://happygitwithr.com/
```

**Comment by aays** [comment]
Thanks for the feedback! Incorporated both comments and created a barebones `lesson.md` file.

**Pull request merged**